content/elections-contest.md

```diff
@@ -0,0 +1,197 @@
+    title = "Find your polling place"
+    emoji = ":round_pushpin:"
+    link = "https://www.pavoterservices.pa.gov/Pages/PollingPlaceInfo.aspx"
+
+    >}}
+
+    {{<sidebar-link
+    title = "Request a mail ballot"
+    emoji = ":email:"
+    link = "https://www.pavoterservices.pa.gov/OnlineAbsenteeApplication/"
+
+    >}}
+
+    {{<sidebar-link
+    title = "Contact your county election office"
+    emoji = ":iphone:"
+    link = "https://www.vote.pa.gov/resources/pages/contact-your-election-officials.aspx"
+    >}}
+  {{</sidebar-links>}}
+  {{<sidebar-links hed="Key Dates" event="true">}}
+    {{<sidebar-link
+    title="Last day to register to vote"
+    date="2023-10-23"
+    link="https://www.pavoterservices.pa.gov/pages/VoterRegistrationApplication.aspx"
+    >}}
+
+    {{<sidebar-link
+    title="Last day to apply for mail ballot"
+    date="2023-10-31"
+    link="https://www.pavoterservices.pa.gov/pages/voterregistrationstatus.aspx"
+    >}}
+
+    {{<sidebar-link
+    title="Election Day"
+    date="2023-11-07"
+    link="https://www.pavoterservices.pa.gov/Pages/PollingPlaceInfo.aspx"
+
+    >}}
+  {{</sidebar-links>}}
+{{</landing-sidebar>}}
+"""
+donate ="""
+{{<donate-slim
+    title="Help Empower Voters. Support Our Vital Election Coverage."
+    cta="Donate"
+>}}
+"""
+
+[[internal-links]]
+title = "Voter Guides"
+link = "guides"
+
+[[internal-links]]
+title = "Issues"
+link = "court-tool"
+
+[[internal-links]]
+title = "Election Event Series"
+link = "events"
+
+[[internal-links]]
+title = "Coverage"
+link = "coverage"
+
+# [[internal-links]]
+# title = "Español"
+# link = "#espan"
+
+[[internal-links]]
+title = "Ask Us"
+link = "contact"
+
+[[internal-links]]
+title = "Credits"
+link = "credits"
+
+
+[[blocks]]
+layout = "events"
+id = "events"
+title = "Spotlight PA voter event series"
+image = "external/f98jyaza883casbzytgtmqpe6c.jpeg"
+
+[[blocks.entries]]
+hed="Court Primer"
+dek="Spotlight PA held a live event about the candidates running for state Supreme Court and more, and what these elections will mean for you."
+date="2023-10-12"
+dateVerbose="Oct. 12, 2023 on Zoom"
+link="https://www.spotlightpa.org/news/2023/09/pennsylvania-election-2023-supreme-commonwealth-superior-event/"
+
+[[blocks.entries]]
+hed="Voter Ready"
+dek="Learn about your voting rights and ask your remaining questions ahead of Election Day. Registration coming soon."
+date="2023-11-02"
+dateVerbose="Nov. 2, 2023 on Zoom"
+link="https://www.spotlightpa.org/news/2023/10/pennsylvania-election-2023-voting-rights-event/"
+
+[[blocks.entries]]
+hed="Result Review"
+dek="And the winners are … Join us to learn about how the judicial election results will influence politics and policy in the years to come. Registration coming soon."
+date="2023-11-16"
+dateVerbose="Nov. 16, 2023 on Zoom"
+link="https://www.spotlightpa.org/news/2023/10/pennsylvania-election-2023-results-event/"
+registration="https://zoom.us/webinar/register/WN_wf_P08sRQF-jRFrnG-T4NA"
++++
```

Comment:
The blocks in here don't really work, do they? Maybe just drop them so we don't try to tweak it and forget why it's not working in 6 months.